src/buildfromsource/FSharp.Compiler.nuget/Microsoft.FSharp.Compiler.nuspec

```diff
@@ -13,53 +15,69 @@
         <projectUrl>$projectUrl$</projectUrl>
         <tags>$tags$</tags>
         <dependencies>
-            <group targetFramework=".NETStandard1.6">
-                <dependency id="Microsoft.NETCore.Platforms" version="1.0.1" />
-                <dependency id="NETStandard.Library" version="1.6.0" />
+            <group targetFramework=".NETStandard2.0">
```

Comment:
I'm pretty sure most if not all of these APIs are available in .NET Standard 2.0. They can be removed. See the FSharp.Core.nuspec file for an example.

Reply:
Yes, they are all available on .NET Standard 2.0. The original trimmed project.json was due to not restoring significantly more packages that necessary; now with the new factoring for the SDK such a step isn't necessary.